### DISCUSSION TITLE
Hotkey Support

### DISCUSSION THREAD

**Discussion by ghost** [MAFINS/MenyooSP#478] [Ideas]
Is it possible to create a hotkey support of some kind?
Like an .ini file to define hotkeys which toggle or activate Menyoo items?

Examples would be:

-Health/Ammo Replenishment
- Player toggles..
- really any GUI menyoo item?

Could be useful in cutting time for testing, increasing time for fun, and even helping out the occasional streamer out there

**Comment by MAFINS** [comment]
It is possible, yes.
Howerver, many mods such as simple native trainer already provide such quick actions.
